src/app/(marketing)/landing/components/landing.module.css

```
@@ -0,0 +1,117 @@
+
+/* Constellation route: draws in once its Reveal wrapper becomes visible */
+.routePath {
+  stroke-dasharray: 1;
+  stroke-dashoffset: 1;
+}
+
+.revealVisible .routePath {
+  animation: drawRoute 1.8s cubic-bezier(0.4, 0, 0.2, 1) 0.2s forwards;
+}
+
+@keyframes drawRoute {
```

Stylelint (17.14.0)

[error] Expected keyframe name "drawRoute" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)